configuration/configOpenShiftDEV.py

```diff
@@ -20,7 +20,12 @@ class Logging(object):
         MAIN_LOG_BASENAME = 'main'
         LOG_FILENAME_EXTENSION = '.log'
         DEFAULT_LOG_LEVEL = logging.INFO
-
+    
+    # Other settings.
+    class General(object):
+        # include any required whitespace.
+        ASGN_FOLDER_PREFIX = 'ASGN: '
```

Comment:
IMHO, this belongs in the `ArcGIS` class (line 44).

Reply:
arcgisUM is designed to map between the samantic needs of the driver and actual calls to ArcGIS.  It should know little about the problem domain.  Folder names are provided by the caller so literals put in folder names should be supplied by the caller and specified in a namespace appropriate to the caller.